### ISSUE TITLE
Error when compiling examples

### ISSUE THREAD

**Issue by MorenoIgor** [alekmaul/pvsneslib#227]
Hi there! I've been trying to tinker around with the samples but all I get is the following error:

> Compiling to .ps ... hello_world.c
/c/pvsneslib/devkitsnes/bin/816-tcc -I/c/pvsneslib/pvsneslib/include -I/c/pvsneslib/devkitsnes/include -I/c/pvsneslib/snes-examples/hello_world -Wall -c src/hello_world.c -o src/hello_world.ps
Assembling ... hello_world.ps
/c/pvsneslib/devkitsnes/bin/816-opt.py src/hello_world.ps >src/hello_world.asp
Cannot openmake: *** [src/hello_world.asm] Error 1
rm src/hello_world.ps

The error occurs if I attempy a clean make or also if I make any modifications to the original source files.

I saw that someone had a similar problem before (https://github.com/alekmaul/pvsneslib/issues/71), but the thread had no info on how to fix the issue. I'm currently running Windows 11, so I think that maybe this is the main issue.

I've followed the instructions on installing the lib - environment variables, PATH, etc. Here is a video of the process, in case it helps: https://youtu.be/i6bCa9MtRX8

I'm not well versed in using makefiles, so I could not for the life of me grasp what was going on, even after messing with it for a day. 

Any hints? Cheers!

**Comment by alekmaul** [comment]
1st question, which version of pvsneslib are you using ? the release one ?
second question, is msys intalled as decribed here ? https://github.com/alekmaul/pvsneslib/wiki/Installation
You issue seems to be something like msys is not installed.

**Comment by MorenoIgor** [comment]
Hey there! Yes, I'm using the latest release. And I've also installed msys as indicated, adding both the root msys folder and msys\bin to the path. Msys starts and runs apparently normally as shown in the video, it just does not go beyond the python phase of the compilation.

While digging a bit further it appears to me that the python script is not executing properly, even though python 3.11 is installed. Is there any extra step to enable python to work with msys?

**Comment by alekmaul** [comment]
sorry, i just checked your video, it's really helpful to did that ;)
It is a strange problem, perhaps a python version?
i'm using python 2.7 not the latest one in the release version (we are going to remove the use of python).
python needs to be in your path, but i think it is the case, is it possible to test with a 2.x version ?.

**Comment by MorenoIgor** [comment]
I got it working! When uninstalling Python to test with an older version I came upon the weird way in which Win11 handles the "python" command - it apparently treats it by default as a shortcut to install it from the app store. Somehow this was firing up even though python was already installed, and I managed to disable that. 

Now it works perfectly, Python 3.11 and all. Thanks for the help! Really excited to write some games. =D
